.github/workflows/build.yaml

```diff
@@ -31,6 +29,11 @@ jobs:
         run: |-
           cargo install cargo-make
 
+      - uses: knox-networks/github-actions-public/.github/actions/setup-protofetch@main
```

Comment:
Should we use `wget` or `curl` to pull in a prebuilt binary?
If we plan to use this change downstream, building `protofetch` will add roughly two minutes to the CI time:
 https://github.com/knox-networks/ssi/actions/runs/6052152827/job/16425074210?pr=145